lib/media/track/remotevideotrack.js

```diff
@@ -4,8 +4,10 @@ const mixinRemoteMediaTrack = require('./remotemediatrack');
 const VideoTrack = require('./videotrack');
 const documentVisibilityMonitor = require('../../util/documentvisibilitymonitor.js');
 const { NullObserver } = require('../../util/nullobserver.js');
+const Timeout = require('../../util/timeout');
 
 const RemoteMediaVideoTrack = mixinRemoteMediaTrack(VideoTrack);
+const TRACK_TURN_OF_DELAY_MS = 50;
```

Comment:
Should we put this in constants.js file? Also, if my understanding is correct, this implementation is essentially debouncing right? I think 50ms is not enough. I can see the turn on/off behavior if I scroll the participants list on/off screen too fast. How about 500ms?

Reply:
I think it makes it easy to read when declared locally, but I am open to moving this to constants


Re: 50ms - We decided on 50ms as we are solving only for  re-layout issue - we are not solving for the case where user may scroll up and down quickly. 

The issue we are solving is - when dominant speaker change in our ahoy app (and any apps that follow similar pattern) the position of dominant speaker video changes, it causes a relayout causing the track to be turned off and on quickly. 
our justification for 50ms is here: https://docs.google.com/document/d/11zgCOIYux9vwlNdzd-smrGgnLQf3x0863N-WQ7zzSq8/edit#heading=h.thlab8wykvo